src/packages/src/xroad/ubuntu/generic/xroad-proxy-ui-api.postinst

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+service xroad-proxy-ui-api restart
```

Comment:
duplicate restart, already handled by #DEBHELPER#

Reply:
will add separate jira ticket for this https://jira.niis.org/browse/XRDDEV-723?focusedCommentId=12716&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-12716 and restesting